IBM_DB/ibm_db/ibm_db_tests/test_currentschema.py

```diff
@@ -0,0 +1,55 @@
+#
+#  Licensed Materials - Property of IBM
+#
+#  (c) Copyright IBM Corp. 2007-2020
+#
+
+from __future__ import print_function
+import sys
+import unittest
+import ibm_db
+import ibm_db_dbi
+import config
+
+class IbmDbTestCase(unittest.TestCase):
+
+    def test_currentschema(self):
+        self.run_default_current_schema()
+        self.run_setup_schema()
+
+    def _assert_current_schema(self, conn, schema_name):
+        query = "SELECT CURRENT_SCHEMA FROM SYSIBM.SYSDUMMY1"
+        try:
+            stmt = ibm_db.exec_immediate(conn, query)
+            res = ibm_db.fetch_tuple(stmt)
+            self.assertEqual(res[0].strip(), schema_name.upper())
+        except:
```

Comment:
I would suggest something slightly different here. I assume the point of the except is to capture some query error or something like it with the db. If that is the case then I would suggest:

```
except Exception as e:
    print("Query failed for: ", e)
    raise
```

It results in the same thing, it just doesn't obscure what the error was in the case that some debugging needs to happen.

Reply:
Checkout https://stackoverflow.com/questions/4990718/about-catching-any-exception

If you insistent your suggestion, create a PR to the origin repo :)